src/pages/events/[eventId]/public-attendees.tsx

```diff
@@ -0,0 +1,12 @@
+import { useRouter } from 'next/router';
+import React from 'react';
+
+import { PublicAttendees } from 'events/components/DetailView/PublicAttendees';
+
+const PublicAttendeesPage = () => {
+  const { query } = useRouter();
+  const eventId = Number(query.eventId);
+  return <PublicAttendees eventId={eventId} />;
+};
+
+export default PublicAttendeesPage;
```

Comment:
So this component would perhaps be deleted when integrated with i.e. a modal?
#352 for example?

Reply:
Yep